lib/mapping_loader.rb

```diff
@@ -20,13 +20,15 @@ def csv_to_mapping(csv)
   mapping = {}
   csv.each do |row|
     # skip rows that don't have a source field
-    next if row['source_field'].nil? || row['source_field'].strip.empty?
+    next if row['source_field'].nil? || row['source_field'].strip.empty? || row['destination_field'].nil? || row['destination_field'].strip.empty?
```

Comment:
This is fine for now, but we'll have to revisit, when we tackle hardcoding values. Only having a destination_field + a value will be legit then.

For readability, I'd suggest moving this to one per line
```suggestion
    next if row['source_field'].nil? ||
            row['source_field'].strip.empty? ||
            row['destination_field'].nil? ||
            row['destination_field'].strip.empty?
```